### ISSUE TITLE
AttributeError: 'EvaluateArguments' object has no attribute 'eval_language'

### ISSUE THREAD

**Issue by slatter666** [codefuse-ai/codefuse-devops-eval#18]
When I try to execute the testing script, I get this error, I guess perheps it's because you forget to update the code or anything? Another question is that should I modify dataset_fp.json? cause it only contains part of the whole dataset

**Comment by david101-hunter** [comment]
No, You only add like this to [here](https://github.com/codefuse-ai/codefuse-devops-eval/blob/test/src/hparams/evaluate_args.py)
```python
    eval_language: str = field(
        default = 'en',
        metadata={"help": "Which language of datasets to evaluate on. default is en, must be one of (en, zh)"}
    )
```
when run, depend on you to choose en or zh. 

![image](https://github.com/user-attachments/assets/b58f62db-e968-424c-a59b-76d6926d0c88)
